src/tables/GSUB.js

```diff
@@ -53,7 +53,7 @@ let GSUBLookup = new r.VersionedStruct('lookupType', {
   7: { // Extension Substitution
     substFormat:   r.uint16,
     lookupType:    r.uint16,   // cannot also be 7
-    extension:     new r.Pointer(r.uint32, GSUBLookup)
```

Comment:
This also doesn look like an eval/new Function removal.

Reply:
@Pomax: true. But it's a fix for modern browser that honor 'let'. The varible GSUBLookup is not defined at that point at time anyway. If converted to 'var' it is undefined.